modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/internal/model/listener/test/UserModelListenerTest.java

```diff
@@ -67,22 +76,27 @@ public void setUp() throws Exception {
 
 		Bundle bundle = FrameworkUtil.getBundle(UserModelListenerTest.class);
 
-		BundleContext bundleContext = bundle.getBundleContext();
-
-		_serviceRegistration = bundleContext.registerService(
-			SystemObjectDefinitionManager.class,
-			new TestSystemObjectDefinitionManager(
-				ObjectEntry.class, _OBJECT_DEFINITION_NAME,
-				StringBundler.concat(
-					"/o/", RandomTestUtil.randomString(), StringPool.SLASH,
-					RandomTestUtil.randomString())),
-			new HashMapDictionary<>());
+		_bundleContext = bundle.getBundleContext();
+
+		_systemObjectDefinitionManagerServiceRegistration =
+			_bundleContext.registerService(
+				SystemObjectDefinitionManager.class,
+				new TestSystemObjectDefinitionManager(
+					ObjectEntry.class, _OBJECT_DEFINITION_NAME,
+					StringBundler.concat(
+						"/o/", RandomTestUtil.randomString(), StringPool.SLASH,
+						RandomTestUtil.randomString())),
+				new HashMapDictionary<>());
 	}
 
 	@After
 	public void tearDown() throws Exception {
-		if (_serviceRegistration != null) {
-			_serviceRegistration.unregister();
+		if (_systemObjectDefinitionManagerServiceRegistration != null) {
+			_systemObjectDefinitionManagerServiceRegistration.unregister();
+		}
+
+		if (_auditMessageProcessorServiceRegistration != null) {
+			_auditMessageProcessorServiceRegistration.unregister();
```

Comment:
@thallescezar please sort this `_auditMessageProcessorServiceRegistration` should come before `_systemObjectDefinitionManagerServiceRegistration`,

Thx!